tests/libconfigmap/configmap.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+package libconfigmap
```

Comment:
Would you add the new file to as many linters as sensible?

Reply:
I added another commit to include tests/libconfigmap. Is this what you referred to?

Reply:
yes, this seems fair. I just expected to be on the same commit that introduced the new file.

Reply:
And please, put a newline between `*/` and `package` :)

Reply:
@Barakmor1 it's a nit, but please notice @fossedihelm 's request for a newline. That's what we have in all package files.

/lgtm

Reply:
> @Barakmor1 it's a nit, but please notice @fossedihelm 's request for a newline. That's what we have in all package files.
> 
> /lgtm

Commenting in a resolved thread will make all addition hidden.
Better open a new one.

Reply:
Done